package.json

```diff
@@ -32,5 +32,8 @@
     "gulp-plumber": "^1.2.0",
     "gulp-uglify": "^3.0.0",
     "pump": "^3.0.0"
+  },
+  "dependencies": {
+    "showdown": "^1.8.6"
```

Comment:
should be a dev dependency?

Reply:
effectively it doesn't matter, but for correctness yes ... just part of the build pipeline, only the html is shipped